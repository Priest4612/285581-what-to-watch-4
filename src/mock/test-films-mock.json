[
	{
		"id": 0,
		"name": "We Need To Talk About Kevin",
		"posterImage": "img/we-need-to-talk-about-kevin.jpg",
		"previewImage": "img/we-need-to-talk-about-kevin.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 4.68,
		"scoresCount": 737,
		"director": "- Дед, а ну, подожди, - сказал я, взяв деда за руку.",
		"starring": [
			"Зак Снайдер",
			"Леонид Гайдай",
			"Дэвид Линч",
			"Пол Верховен",
			"Ридли Скотт",
			"Джеймс Кэмерон",
			"Альфред Хичкок",
			"Джим Джармуш"
		],
		"runTime": 77,
		"genre": "Сергей Безруков",
		"released": 1951,
		"isFavorite": false
	},
	{
		"id": 1,
		"name": "Seven Years In Tibet",
		"posterImage": "img/seven-years-in-tibet.jpg",
		"previewImage": "img/seven-years-in-tibet.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 8.17,
		"scoresCount": 374,
		"director": "Я мучился, потому что мне показалось, что с нею необходимо говорить, и тревожился, что я не вымолвлю ни одного слова, а она уйдет, и я никогда ее более не увижу...",
		"starring": [
			"Акира Куросава",
			"Ингмар Бергман",
			"Дэвид Финчер",
			"Роман Полански",
			"Мэл Гибсон",
			"Мартин Скорсезе",
			"Люк Бессон",
			"Чарли Чаплин",
			"Сидни Люмет"
		],
		"runTime": 110,
		"genre": "Дженнифер Энистон",
		"released": 1901,
		"isFavorite": false
	},
	{
		"id": 2,
		"name": "Moonrise Kingdom",
		"posterImage": "img/moonrise-kingdom.jpg",
		"previewImage": "img/moonrise-kingdom.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 6.75,
		"scoresCount": 947,
		"director": "И меня поразила не столько ее красота, сколько необыкновенное, никем не виданное одиночество в глазах!",
		"starring": [
			"Джон МакТирнан",
			"Акира Куросава",
			"Фрэнсис Форд Коппола",
			"Питер Джексон",
			"Альфонсо Куарон",
			"Стивен Спилберг"
		],
		"runTime": 95,
		"genre": "Дженсен Эклс",
		"released": 2000,
		"isFavorite": false
	},
	{
		"id": 3,
		"name": "What We Do In The Shadows",
		"posterImage": "img/what-we-do-in-the-shadows.jpg",
		"previewImage": "img/what-we-do-in-the-shadows.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 8.03,
		"scoresCount": 684,
		"director": "- Мы… мы… 45 лет вместе… Она заболела… Я не мог её оставить сегодня без подарка, - тихо сказал дед, спасибо тебе...",
		"starring": [
			"Федерико Феллини",
			"Люк Бессон",
			"Чарли Чаплин",
			"Альфонсо Куарон",
			"Мартин Скорсезе",
			"Клинт Иствуд",
			"Уэс Андерсон"
		],
		"runTime": 120,
		"genre": "Тэйлор Лотнер",
		"released": 1964,
		"isFavorite": true
	},
	{
		"id": 4,
		"name": "Mindhunter",
		"posterImage": "img/mindhunter.jpg",
		"previewImage": "img/mindhunter.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 5.78,
		"scoresCount": 203,
		"director": "И тут я вспомнил, что я не купил себе цветы.",
		"starring": [
			"Дэвид Кроненберг",
			"Даррен Аронофски",
			"Хаяо Миядзаки",
			"Ларс фон Триер",
			"Фрэнсис Форд Коппола",
			"Ингмар Бергман"
		],
		"runTime": 7,
		"genre": "Шарлиз Терон",
		"released": 1967,
		"isFavorite": true
	},
	{
		"id": 5,
		"name": "No Country For Old Men",
		"posterImage": "img/no-country-for-old-men.jpg",
		"previewImage": "img/no-country-for-old-men.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 5.07,
		"scoresCount": 333,
		"director": "Мы шли по кривому, скучному переулку безмолвно, я по одной стороне, а она по другой. И не было, вообразите, в переулке ни души.",
		"starring": [
			"Джоэл и Итан Коэны",
			"Джон МакТирнан",
			"Джим Джармуш",
			"Гай Ричи",
			"Альфонсо Куарон",
			"Акира Куросава",
			"Джеймс Кэмерон"
		],
		"runTime": 30,
		"genre": "Джордж Клуни",
		"released": 2018,
		"isFavorite": false
	},
	{
		"id": 6,
		"name": "Moonrise Kingdom",
		"posterImage": "img/moonrise-kingdom.jpg",
		"previewImage": "img/moonrise-kingdom.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 8.03,
		"scoresCount": 513,
		"director": "Я бежал, даже не понимая, куда бегу. Слёзы сами текли из моих глаз…",
		"starring": [
			"Жан-Люк Годар"
		],
		"runTime": 169,
		"genre": "Шон Пенн",
		"released": 1964,
		"isFavorite": false
	},
	{
		"id": 7,
		"name": "Dardjeeling Limited",
		"posterImage": "img/dardjeeling-limited.jpg",
		"previewImage": "img/dardjeeling-limited.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 8.67,
		"scoresCount": 274,
		"director": "- А ты сынок подожди, она отошла ненадолго, щас вернётся, - сказала тётка, торговавшая по соседству солёными огурцами.",
		"starring": [
			"Билли Уайлдер",
			"Акира Куросава",
			"Зак Снайдер",
			"Карл Теодор Дрейер",
			"Чарли Чаплин",
			"Ингмар Бергман",
			"Стэнли Кубрик",
			"Альфонсо Куарон",
			"Ларс фон Триер"
		],
		"runTime": 35,
		"genre": "Вин Дизель",
		"released": 2017,
		"isFavorite": true
	},
	{
		"id": 8,
		"name": "Snatch",
		"posterImage": "img/snatch.jpg",
		"previewImage": "img/snatch.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 8.03,
		"scoresCount": 912,
		"director": "Дед мотал головой в знак отказа, и второй рукой прикрывал свою поломанную ветку.",
		"starring": [
			"Джон МакТирнан",
			"Джим Джармуш",
			"Кристофер Нолан",
			"Стивен Спилберг",
			"Стэнли Кубрик",
			"Даррен Аронофски",
			"Майкл Манн",
			"Роман Полански"
		],
		"runTime": 122,
		"genre": "Харрисон Форд",
		"released": 1952,
		"isFavorite": false
	},
	{
		"id": 9,
		"name": "Revenant",
		"posterImage": "img/revenant.jpg",
		"previewImage": "img/revenant.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 5.48,
		"scoresCount": 70,
		"director": "Деда трусило от холода, его руки были синего цвета… Его очень трусило, но он стоял на ветру и ждал.",
		"starring": [
			"Дэвид Кроненберг",
			"Альфонсо Куарон",
			"Чарли Чаплин",
			"Ингмар Бергман",
			"Квентин Тарантино",
			"Леонид Гайдай"
		],
		"runTime": 120,
		"genre": "АНДРЕЙ МИРОНОВ",
		"released": 1950,
		"isFavorite": false
	},
	{
		"id": 10,
		"name": "Seven Years In Tibet",
		"posterImage": "img/seven-years-in-tibet.jpg",
		"previewImage": "img/seven-years-in-tibet.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 10.32,
		"scoresCount": 310,
		"director": "Я бы не сказал, что погода была весенней. Ледяной ветер так и норовил забраться под куртку. Подняв воротник и опустив в него как можно ниже голову, я приближался к базару. Я ещё за неделю до этого решил: никаких роз, только весенние цветы… праздник же весенний.",
		"starring": [
			"Гай Ричи",
			"Кристофер Нолан",
			"Мартин Скорсезе",
			"Сидни Люмет",
			"Фрэнсис Форд Коппола",
			"Билли Уайлдер",
			"Роман Полански",
			"Джоэл и Итан Коэны",
			"Стивен Спилберг"
		],
		"runTime": 62,
		"genre": "Брюс Уиллис",
		"released": 1974,
		"isFavorite": false
	},
	{
		"id": 11,
		"name": "The Grand Budapest Hotel Poster",
		"posterImage": "img/the-grand-budapest-hotel-poster.jpg",
		"previewImage": "img/the-grand-budapest-hotel-poster.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 1.41,
		"scoresCount": 362,
		"director": "Она нагнулась к корзине, долго в ней ковырялась…",
		"starring": [
			"Ингмар Бергман",
			"Билли Уайлдер",
			"Альфред Хичкок",
			"Пол Верховен",
			"Клинт Иствуд",
			"Мэл Гибсон",
			"Пол Томас Андерсон"
		],
		"runTime": 135,
		"genre": "Джейсон Стэтхэм",
		"released": 1914,
		"isFavorite": true
	},
	{
		"id": 12,
		"name": "Fantastic Beasts The Crimes Of Grindelwald",
		"posterImage": "img/fantastic-beasts-the-crimes-of-grindelwald.jpg",
		"previewImage": "img/fantastic-beasts-the-crimes-of-grindelwald.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 8.7,
		"scoresCount": 725,
		"director": "Она нагнулась к корзине, долго в ней ковырялась…",
		"starring": [
			"Вуди Аллен"
		],
		"runTime": 28,
		"genre": "ГЕОРГИЙ ВИЦИН",
		"released": 1952,
		"isFavorite": true
	},
	{
		"id": 13,
		"name": "Macbeth",
		"posterImage": "img/macbeth.jpg",
		"previewImage": "img/macbeth.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 7.77,
		"scoresCount": 918,
		"director": "- Ты, курица тупая, сколько стоит твоё ведро, отвечай быстро и внятно, что бы я не напрягал слух, - еле слышно, но очень понятно прошипел я.",
		"starring": [
			"Ларс фон Триер",
			"Ингмар Бергман",
			"Клинт Иствуд",
			"Джим Джармуш",
			"Тим Бёртон",
			"Фрэнк Дарабонт",
			"Акира Куросава"
		],
		"runTime": 130,
		"genre": "Леонардо ДиКаприо",
		"released": 1944,
		"isFavorite": false
	},
	{
		"id": 14,
		"name": "What We Do In The Shadows",
		"posterImage": "img/what-we-do-in-the-shadows.jpg",
		"previewImage": "img/what-we-do-in-the-shadows.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 5.02,
		"scoresCount": 679,
		"director": "Деда трусило от холода как лист бумаги на ветру.",
		"starring": [
			"Альфред Хичкок",
			"Дэвид Финчер",
			"Карл Теодор Дрейер",
			"Вуди Аллен",
			"Алехандро Гонсалес Иньярриту",
			"Альфонсо Куарон"
		],
		"runTime": 57,
		"genre": "Антонио Бандерас",
		"released": 1971,
		"isFavorite": false
	},
	{
		"id": 15,
		"name": "Midnight Special",
		"posterImage": "img/midnight-special.jpg",
		"previewImage": "img/midnight-special.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 4.61,
		"scoresCount": 124,
		"director": "Напротив меня стоял старик.",
		"starring": [
			"Чарли Чаплин",
			"Стэнли Кубрик",
			"Дени Вильнёв",
			"Мэл Гибсон",
			"Гас Ван Сент",
			"Хаяо Миядзаки",
			"Квентин Тарантино",
			"Пол Верховен",
			"Алехандро Гонсалес Иньярриту",
			"Пол Томас Андерсон"
		],
		"runTime": 25,
		"genre": "Джулия Робертс",
		"released": 2003,
		"isFavorite": false
	},
	{
		"id": 16,
		"name": "Johnny English",
		"posterImage": "img/johnny-english.jpg",
		"previewImage": "img/johnny-english.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 5.94,
		"scoresCount": 660,
		"director": "Видимо, в моих глазах было что-то такое, что продавщица как-то побледнела и даже уменьшилась в росте. Она просто смотрела на меня как мышь на удава и молчала.",
		"starring": [
			"Фрэнсис Форд Коппола",
			"Мэл Гибсон",
			"Гас Ван Сент",
			"Чарли Чаплин",
			"Клинт Иствуд",
			"Пол Томас Андерсон",
			"Майкл Манн",
			"Андрей Тарковский",
			"Альфред Хичкок"
		],
		"runTime": 59,
		"genre": "Тиль Швайгер",
		"released": 1932,
		"isFavorite": false
	},
	{
		"id": 17,
		"name": "Moonrise Kingdom",
		"posterImage": "img/moonrise-kingdom.jpg",
		"previewImage": "img/moonrise-kingdom.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 2.17,
		"scoresCount": 297,
		"director": "В синей от холода руке деда я увидел ветку мимозы, она была сломана посередине.",
		"starring": [
			"Уэс Андерсон",
			"Карл Теодор Дрейер",
			"Алехандро Гонсалес Иньярриту",
			"Хаяо Миядзаки",
			"Питер Джексон",
			"Кристофер Нолан",
			"Люк Бессон"
		],
		"runTime": 54,
		"genre": "Анджелина Джоли",
		"released": 2014,
		"isFavorite": true
	},
	{
		"id": 18,
		"name": "Macbeth",
		"posterImage": "img/macbeth.jpg",
		"previewImage": "img/macbeth.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 2.93,
		"scoresCount": 293,
		"director": "В синей от холода руке деда я увидел ветку мимозы, она была сломана посередине.",
		"starring": [
			"Кристофер Нолан",
			"Билли Уайлдер",
			"Ридли Скотт",
			"Жан-Люк Годар",
			"Дэвид Кроненберг"
		],
		"runTime": 114,
		"genre": "Нина Добрев",
		"released": 1981,
		"isFavorite": true
	},
	{
		"id": 19,
		"name": "The Grand Budapest Hotel Poster",
		"posterImage": "img/the-grand-budapest-hotel-poster.jpg",
		"previewImage": "img/the-grand-budapest-hotel-poster.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 10.02,
		"scoresCount": 873,
		"director": "И тут я вспомнил, что я не купил себе цветы.",
		"starring": [
			"Пол Томас Андерсон",
			"Федерико Феллини",
			"Даррен Аронофски",
			"Ридли Скотт",
			"Гай Ричи",
			"Серджио Леоне",
			"Стивен Спилберг"
		],
		"runTime": 177,
		"genre": "Орландо Блум",
		"released": 1984,
		"isFavorite": true
	},
	{
		"id": 20,
		"name": "Orlando",
		"posterImage": "img/orlando.jpg",
		"previewImage": "img/orlando.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 3.01,
		"scoresCount": 820,
		"director": "Дед мотал головой в знак отказа, и второй рукой прикрывал свою поломанную ветку.",
		"starring": [
			"Федерико Феллини",
			"Джон Карпентер",
			"Стэнли Кубрик",
			"Мартин Скорсезе",
			"Даррен Аронофски",
			"Альфред Хичкок",
			"Леонид Гайдай"
		],
		"runTime": 152,
		"genre": "Джет Ли",
		"released": 2001,
		"isFavorite": false
	},
	{
		"id": 21,
		"name": "Midnight Special",
		"posterImage": "img/midnight-special.jpg",
		"previewImage": "img/midnight-special.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 1.08,
		"scoresCount": 916,
		"director": "Деда трусило от холода, его руки были синего цвета… Его очень трусило, но он стоял на ветру и ждал.",
		"starring": [
			"Леонид Гайдай",
			"Чарли Чаплин",
			"Дэвид Финчер",
			"Дени Вильнёв",
			"Ларс фон Триер"
		],
		"runTime": 104,
		"genre": "Шон Пенн",
		"released": 1926,
		"isFavorite": true
	},
	{
		"id": 22,
		"name": "Pulp Fiction",
		"posterImage": "img/pulp-fiction.jpg",
		"previewImage": "img/pulp-fiction.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 1.17,
		"scoresCount": 570,
		"director": "Какой-то нехороший комок подкатил к моему горлу.",
		"starring": [
			"Акира Куросава",
			"Гас Ван Сент",
			"Джон МакТирнан",
			"Мэл Гибсон",
			"Карл Теодор Дрейер",
			"Чарли Чаплин",
			"Фрэнсис Форд Коппола",
			"Даррен Аронофски",
			"Ларс фон Триер"
		],
		"runTime": 14,
		"genre": "Уиллем Дефо",
		"released": 1929,
		"isFavorite": false
	},
	{
		"id": 23,
		"name": "Dardjeeling Limited",
		"posterImage": "img/dardjeeling-limited.jpg",
		"previewImage": "img/dardjeeling-limited.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 1.98,
		"scoresCount": 976,
		"director": "- Хозяюшка, у меня есть три рубля, может, найдёшь для меня веточку на три рубля? - как-то очень тихо спросил дед.",
		"starring": [
			"Роман Полански",
			"Тим Бёртон",
			"Сидни Люмет",
			"Пол Верховен",
			"Питер Джексон",
			"Даррен Аронофски",
			"Гай Ричи"
		],
		"runTime": 163,
		"genre": "Джон Траволта",
		"released": 1920,
		"isFavorite": true
	},
	{
		"id": 24,
		"name": "Seven Years In Tibet",
		"posterImage": "img/seven-years-in-tibet.jpg",
		"previewImage": "img/seven-years-in-tibet.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 3,
		"scoresCount": 123,
		"director": "Деда трусило от холода как лист бумаги на ветру.",
		"starring": [
			"Дени Вильнёв",
			"Альфред Хичкок",
			"Ингмар Бергман"
		],
		"runTime": 21,
		"genre": "Пенелопа Крус",
		"released": 2015,
		"isFavorite": true
	},
	{
		"id": 25,
		"name": "Bohemian Rhapsody",
		"posterImage": "img/bohemian-rhapsody.jpg",
		"previewImage": "img/bohemian-rhapsody.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 5.66,
		"scoresCount": 351,
		"director": "- Так, а ну вали от сюдава алкаш, попрошайничать надумал, давай вали, а то… - прорычала продавщица на деда.",
		"starring": [
			"Гай Ричи",
			"Вуди Аллен",
			"Пол Верховен",
			"Альфонсо Куарон",
			"Хаяо Миядзаки",
			"Джим Джармуш",
			"Андрей Тарковский",
			"Мартин Скорсезе",
			"Фрэнк Дарабонт"
		],
		"runTime": 88,
		"genre": "Кейт Бекинсейл",
		"released": 1909,
		"isFavorite": true
	},
	{
		"id": 26,
		"name": "Midnight Special",
		"posterImage": "img/midnight-special.jpg",
		"previewImage": "img/midnight-special.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 8.57,
		"scoresCount": 79,
		"director": "Черт их знает, как их зовут, но они первые почему-то появляются в Москве.",
		"starring": [
			"Альфред Хичкок",
			"Чарли Чаплин",
			"Джим Джармуш",
			"Джеймс Кэмерон",
			"Тим Бёртон",
			"Стивен Спилберг",
			"Питер Джексон",
			"Дени Вильнёв",
			"Билли Уайлдер",
			"Карл Теодор Дрейер"
		],
		"runTime": 168,
		"genre": "Дженнифер Лопез",
		"released": 1963,
		"isFavorite": false
	},
	{
		"id": 27,
		"name": "Revenant",
		"posterImage": "img/revenant.jpg",
		"previewImage": "img/revenant.jpg",
		"backgroundImage": "img/bg-the-grand-budapest-hotel.jpg",
		"backgroundColors": "#ffffff",
		"videoLink": "bg-the-grand-budapest-hotel.jpg",
		"previewVideoLink": "bg-the-grand-budapest-hotel.jpg",
		"description": [],
		"rating": 8.78,
		"scoresCount": 153,
		"director": "- На, отец, бери, и иди поздравляй свою жену, - сказал я.",
		"starring": [
			"Питер Джексон",
			"Гас Ван Сент",
			"Сидни Люмет"
		],
		"runTime": 149,
		"genre": "Джордж Клуни",
		"released": 1914,
		"isFavorite": true
	}
]