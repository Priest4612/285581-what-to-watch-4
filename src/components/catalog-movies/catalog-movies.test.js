import React from 'react';
import renderer from 'react-test-renderer';

import CatalogMovies from './catalog-movies.jsx';
import mock from '../../mocks/test-films-mock.json';


it(`Render CatalogMovies`, () => {
  const tree = renderer
    .create(<CatalogMovies
      movies={mock.splice(3, 7)}
      onMouseEnterCard={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
