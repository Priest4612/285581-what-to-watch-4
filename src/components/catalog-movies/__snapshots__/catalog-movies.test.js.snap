// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render CatalogMovies 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="What We Do In The Shadows"
        height="175"
        src="img/what-we-do-in-the-shadows.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        What We Do In The Shadows
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="No Country For Old Men"
        height="175"
        src="img/no-country-for-old-men.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        No Country For Old Men
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Seven Years In Tibet"
        height="175"
        src="img/seven-years-in-tibet.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Seven Years In Tibet
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Aviator"
        height="175"
        src="img/aviator.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Aviator
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Snatch"
        height="175"
        src="img/snatch.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Snatch
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Revenant"
        height="175"
        src="img/revenant.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Revenant
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Dardjeeling Limited"
        height="175"
        src="img/dardjeeling-limited.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Dardjeeling Limited
      </a>
    </h3>
  </article>
</div>
`;
